pkg/kubelet/events/event.go

```diff
@@ -34,6 +34,9 @@ const (
 	FailedToCreatePodContainer     = "FailedCreatePodContainer"
 	FailedToMakePodDataDirectories = "Failed"
 	NetworkNotReady                = "NetworkNotReady"
+	StartingVerticalScalingPod     = "StartingVerticalScalingPod"
```

Comment:
Please rename these to BeginPodResize, FailedPodResize, and CompletedPodResize. In hindsight, the feature-gate should have been named InPlacePodResize which better describes what's happening, and InPlacePodVerticalScaling is bit of a mouthful.